audformat/core/utils.py

```diff
@@ -474,32 +483,38 @@ def expand_file_path(
         index: pd.Index,
         root: str,
 ) -> pd.Index:
-    r"""Expand relative path in index with root.
+    r"""Expand path in index with root.
+
+    It applies :func:`os.path.normpath`
```

Comment:
Since the result of the operation can include invalid paths, maybe we should not apply any processing to `root` but simply append whatever the user provides?

Reply:
One problem here is that this mean the user provides `root='./a'` and the index contains `f1` and we end up with
`./af1` which is not what we want. So I think doing some processing with `root` is fine as it also adds no extra processing costs.

Reply:
You are right we should at least ensure root ends on `/`, so we anyway have to process it. So calling `normpath()` should be fine, too.

Reply:
One problem with `normpath` might be:

> On Windows, it converts forward slashes to backward slashes.

I'm not sure if `audeer.path()` that we used before also is doing this?

Reply:
I think it is the same behavior as in `audeer.path()` we are calling `abspath`:

![image](https://user-images.githubusercontent.com/173624/232043619-1cfcc52f-0614-4c7b-9192-6b5c219bd214.png)

Reply:
Sorry, I was wrong here :(

We are calling `os.path.realpath(os.path.expanduser(path))` in `audeer.path()`.
So switching to `normpath()` does indeed change the behavior on Windows.

 Do we want this to happen?

Reply:
Or maybe not, honestly I don't have a clue how these functions behave under Windows.

Reply:
Me neither and I already merged.

Reply:
Don't worry I checked at https://github.com/audeering/audeer/pull/97
and `audeer.path()` does indeed change `/` to `\`: 

![image](https://user-images.githubusercontent.com/173624/232047402-e3e0e730-cd75-4883-8933-08975dbcd963.png)

So we don't added any breaking change here.